### PR TITLE
Support SHA256 based plugins

### PR DESCRIPTION
**Pull request by ryu-s-r**
Support SHA256 based plugins

### CODE REVIEW

File: src/main/java/com/apkspectrum/plugin/PackageSearcherLinker.java [hunk 1]
```diff
@@ -3,12 +3,14 @@
 import java.awt.Desktop;
 import java.io.UnsupportedEncodingException;
 import java.net.URI;
-import java.net.URLEncoder;
+import java.net.URLEncoder;;
```
Comment (sunggyu-kam) [inline src/main/java/com/apkspectrum/plugin/PackageSearcherLinker.java]: Duplicate ";".

File: src/main/java/com/apkspectrum/plugin/PackageSearcherLinker.java [hunk 2]
```diff
@@ -37,6 +39,10 @@ public void launch() {
 			name = ApkInfoHelper.getResourceValue(labels,
 					getPreferLangForAppName());
 			break;
+        	case SEARCHER_TYPE_PACKAGE_HASH :
```
Comment (sunggyu-kam) [inline src/main/java/com/apkspectrum/plugin/PackageSearcherLinker.java]: Please set the indentation to tab.
I plan to change the indentation method to spaces in the future, but
Please set it as a tab until it changes.

**Review by sunggyu-kam — CHANGES_REQUESTED** [review]

File: src/main/java/com/apkspectrum/plugin/PackageSearcherLinker.java (hunk 2 above)
Comment (ryu-s-r) [reply src/main/java/com/apkspectrum/plugin/PackageSearcherLinker.java]: Oops, I didn't check the indentation. Thanks for your correction.

File: src/main/java/com/apkspectrum/plugin/PackageSearcherLinker.java [hunk 3]
```diff
@@ -40,9 +40,9 @@ public void launch() {
 					getPreferLangForAppName());
 			break;
         	case SEARCHER_TYPE_PACKAGE_HASH :
```
Comment (sunggyu-kam) [inline src/main/java/com/apkspectrum/plugin/PackageSearcherLinker.java]: Line 42 is not corrected.

**Review by sunggyu-kam — CHANGES_REQUESTED** [review]

**Pull request merged**